application/src/main/java/de/wuespace/telestion/project/daedalus2/database/MongoDatabaseService.java

```diff
@@ -47,11 +47,18 @@
 	/**
 	 * This constructor supplies default options.
 	 *
-	 * @param dbName     the name of the local running database
-	 * @param dbPoolName the name of the database pool
+	 * @param host       The host the MongoDB instance is running. Defaults to {@code 127.0.0.1}.
```

Comment:
I'm pretty sure the constructor doesn't default to some value.

Reply:
Of course!
Fixed in https://github.com/wuespace/telestion-project-daedalus2/pull/123/commits/af0fe65a6b7b9542a91a829c2215bcfa4664a5c7.